novawallet/Modules/ManualBackupWalletList/ManualBackupWalletListProtocols.swift

```diff
@@ -0,0 +1,11 @@
+protocol ManualBackupWalletListPresenterProtocol: WalletsListPresenterProtocol {
+    func title() -> String
```

Comment:
I think it is too complicated for concrete implementation of the list. I would just call R from View Controller to get the title.